theano/gof/opt.py

```diff
@@ -2623,16 +2623,16 @@ def check_stack_trace(f_or_fgraph, ops_to_check='last', bug_print='raise'):
     Parameters
     ----------
     f_or_fgraph: theano.compile.function_module.Function or
-                theano.gof.fg.FunctionGraph
+          theano.gof.fg.FunctionGraph
         The compiled function or the function graph to be analysed.
-    ops_to_check: theano.gof.Op or tuple of theano.gof.Op or a string or a
+    ops_to_check: theano.gof.Op or tuple/list of theano.gof.Op or a string or a
         function returning a boolean and taking as input a theano.gof.Op.
         - if ops_to_check is a string, it should be either 'last' or 'all'.
           'last' will check only the last op of the graph while 'all' will
           check all the ops of the graph.
-        - if ops_to_check is an op or a tuple of ops, the function will check
-          that all the outputs of their occurrences in the graph have a stack
-          trace.
+        - if ops_to_check is an op or a tuple/list of ops, the function will
+          check that all the outputs of their occurrences in the graph have a
+          stack trace.
```

Comment:
I've corrected it, and I also check ``isinstance(node.op.scalar_op, ops_to_check)``.